pkg/health/health_order.go

```diff
@@ -18,13 +18,13 @@ func (s SortOrder) Swap(i, j int) {
 
 func (s SortOrder) Less(i, j int) bool {
 	iHealth := Unknown
-	if res, ok := s.Health[s.Nodes[i]]; ok && len(res) > 0 {
-		_, iHealth = FindWorst(res)
+	if res, ok := s.Health[s.Nodes[i]]; ok && &res != nil {
```

Comment:
hmm I need to think about this one for a bit. Can taking the address of a variable result in nil? I might need an explanation on this one.